docs/source/api/email.md

```diff
@@ -83,6 +83,27 @@ Meteor.call(
   'This is a test of Email.send.'
 );
 ```
+{% apibox "Email.sendAsync" %}
+
+`sendAsync` only works on the server. It has the same behavior as `Email.send`, but returns a Promise.
+
+```js
+// Server: Define a method that the client can call.
+Meteor.methods({
+  sendEmail(to, from, subject, text) {
+    // Make sure that all arguments are strings.
+    check([to, from, subject, text], [String]);
+
+    // Let other method calls from the same client start running, without
+    // waiting for the email sending to complete.
+    this.unblock();
+
+    Email.sendAsync({ to, from, subject, text }).catch(err => {
```

Comment:
It'd be good to either add `return` or `await` here, to make sure the method returns only when this call is finished.